src/http/RegisteredHeadersHash.cci

```diff
@@ -363,8 +363,8 @@ static const struct HeaderTableRecord HttpHeaderDefinitionsTable[] =
     {"Last-Modified", Http::HdrType::LAST_MODIFIED, Http::HdrFieldType::ftDate_1123, HdrKind::EntityHeader}
 };
 
-const struct HeaderTableRecord *
-HttpHeaderHashTable::lookup (const char *str, size_t len)
+const class HeaderTableRecord *
+    HttpHeaderHashTable::lookup (const char *str, size_t len)
```

Comment:
I do not know why my astyle v2.04 indents this line while the official (probably) does not.